docs/getting-started.md

```diff
@@ -157,6 +157,22 @@ You can check out [this example project](https://github.com/henrikra/nativeNavig
 
 Remember that you need to wrap each screen that you use in your app with `gestureHandlerRootHOC` as with native navigation libraries each screen maps to a separate root view. It will not be enough to wrap the main screen only.
 
+### For library authors
+
+If you're using gesture handler in your component library, you may want to wrap your library's code in the `GestureHandlerRootView` component. This will avoid extra configuration in `MainActivity.java` for the user.
```

Comment:
We may want to mention here that if you use props like `shouldCancelWhenOutside`, `simultaneousHandlers` or `waitFor` in the handlers they need to be mounted under a single root component. So it is important that GH root view is as close to the actual root view as possible.

Reply:
Thanks! Will update